omod/src/main/webapp/admin/users/userForm.jsp

```diff
@@ -103,9 +103,31 @@
 			</tr>
 		</table>
 	</fieldset>	
-	
 	<br/>
 	
+    <fieldset>
+        <legend><openmrs:message code="legacyui.manageuser.providerAccount"/></legend>
+        <c:if test="${isProvider}">
+            <openmrs:message code="legacyui.manageuser.providerIdentfier"/>
+            <c:forEach items="${providerList}" var="currentProvider" varStatus="loop">   
+                <c:if test="${empty currentProvider.identifier}">
```

Comment:
Hello, my name is Wyclif.

On Wed, Mar 2, 2016 at 7:45 PM, Shreyans Sheth <notifications@github.com>
wrote:

> In omod/src/main/webapp/admin/users/userForm.jsp
> <https://github.com/openmrs/openmrs-module-legacyui/pull/41#discussion_r54712771>
> :
>
> >  	<br/>
> >  	
> > +    <fieldset>
> > +        <legend><openmrs:message code="legacyui.manageuser.providerAccount"/></legend>
> > +        <c:if test="${isProvider}">
> > +            <openmrs:message code="legacyui.manageuser.providerIdentfier"/>
> > +            <c:forEach items="${providerList}" var="currentProvider" varStatus="loop">
> > +                <c:if test="${empty currentProvider.identifier}">
>
> empty checks for null values and prints the value accordingly. I believe
> this is how you wanted it done in the JSP :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openmrs/openmrs-module-legacyui/pull/41/files#r54712771>
> .
>